gfx/sfml_canvas.cpp

```diff
@@ -81,10 +81,13 @@ constexpr std::string_view border_fragment_shader{
         }
         )"sv};
 
-auto get_font_dir_iterator(std::filesystem::path const &path) try {
-    return std::filesystem::recursive_directory_iterator(path);
-} catch (std::filesystem::filesystem_error const &) {
-    return std::filesystem::recursive_directory_iterator();
+std::filesystem::recursive_directory_iterator get_font_dir_iterator(std::filesystem::path const &path) {
+    std::error_code errc;
```

Comment:
Should `<system_error>` be included?

Reply:
It would make me and include-what-you-use happier, so added it! Good catch. :P